docs/KinesisSource-streamingsource.md

```diff
@@ -21,20 +21,20 @@ numbers in DynamoDB table. (Macro-enabled)
 
 **region:** Valid Kinesis region URL eg. ap-south-1. Default value is us-east-1.
 
-**endpointUrl:** Valid Kinesis endpoint URL eg. Kinesis.us-east-1.amazonaws.com. (Macro-enabled)
+**endpointUrl:** Valid Kinesis endpoint URL eg. <protocol>://kinesis.us-east-1.amazonaws.com. For more information on
```

Comment:
while we're fixing things, can you also switch these to use the UI display names instead of the backend names? UI names should also be capitalized ('Kinesis Endpoint URL' instead of 'Kinesis endpoint url')

could you also arrange them so that they are in the same order that they appear in the UI?

Reply:
sure